scripts/vagrant/Vagrantfile

```diff
@@ -4,7 +4,13 @@
 Vagrant.configure("2") do |config|
 
   config.vm.synced_folder "../..", "/go/src/github.com/ligato/networkservicemesh"
+  config.vm.synced_folder ".", "/vagrant"
+  config.vm.allowed_synced_folder_types = [:virtualbox, :sshfs]
```

Comment:
Will this also work for VMWare?

Reply:
I don't know, maybe @nickolaev can answer

Reply:
Well `sshfs` should work on "any"® provider.

Reply:
@nickolaev Yeah... but poorly I would expect...